PlayerControls/sources/ContentControlsViewController+Prism.swift

```diff
@@ -0,0 +1,249 @@
+//  Copyright © 2016 One by Aol : Publishers. All rights reserved.
+
+import Foundation
+
+/// Prism extension for custom content video controls
+public extension ContentControlsViewController.Props {
+    public var noPlayer: Bool {
```

Comment:
IMO, cases without associated values can be better expressed with `isNoPlayer` semantic.